### PR TITLE
RED-343 adds types for blacklisted sender addresses

### PR DESCRIPTION
**Pull request by danielemiliogarcia**
## Description

adds types for blacklisted sender addresses

### CODE REVIEW

File: packages/redshift-types/src/constants/general.ts [hunk 1]
```diff
@@ -202,6 +203,7 @@ export namespace Ws {
     MAKER_QUOTE = 'makerQuote',
     PAYMENT_RESULT = 'paymentResult',
     ACTIVE_CONFIGURATION = 'activeConfiguration',
+    ADDRESS_TED = 'addressBlacklisted',
```
Comment (cavanmflynn) [inline packages/redshift-types/src/constants/general.ts]: `ADDRESS_TED` ->  `ADDRESS_BLACKLISTED`

Comment (danielemiliogarcia) [reply packages/redshift-types/src/constants/general.ts]: woop which was that? hahaha thx

**Comment by cavanmflynn** [comment]
Just realized that `UserSwapState` should be updated as well:

```typescript
export enum UserSwapState {
  WAITING_FOR_FUNDING_TX = 'WaitingForFundingTx',
  WAITING_FOR_FUNDING_TX_CONFIRMATION = 'WaitingForFundingTxConfirmation',
  WAITING_FOR_ADDITIONAL_FUNDING_TX_CONFIRMATION = 'WaitingForAdditionalFundingTxConfirmation',
  PARTIALLY_FUNDED = 'PartiallyFunded',
  FUNDED = 'Funded',

  COMPLETE = 'Complete', // Invoice has been paid

  WAITING_FOR_REFUND_TX = 'WaitingForRefundTx',
  ADDRESS_BLACKLISTED_WAITING_FOR_REFUND_TX = 'AddressBlacklistedWaitingForRefundTx',
  WAITING_FOR_REFUND_TX_CONFIRMATION = 'WaitingForRefundTxConfirmation',
  REFUNDED = 'Refunded',

  FUND_WINDOW_ELAPSED = 'FundWindowElapsed', // The fund window has elapsed
}
```

File: packages/redshift-types/src/constants/general.ts [hunk 2]
```diff
@@ -37,12 +38,18 @@ export enum UserSwapState {
...
 
   WAITING_FOR_REFUND_TX = 'WaitingForRefundTx',
+  ADDRESS_BLACKLISTED_WAITING_FOR_REFUND_TX = 'AddressBlacklistedWaitingForRefundTx',
   WAITING_FOR_REFUND_TX_CONFIRMATION = 'WaitingForRefundTxConfirmation',
   REFUNDED = 'Refunded',
 
   FUND_WINDOW_ELAPSED = 'FundWindowElapsed', // The fund window has elapsed
 }
 
+export enum PaymentFailedReason {
```
Comment (cavanmflynn) [inline packages/redshift-types/src/constants/general.ts]: Add JSDoc comment

Comment (danielemiliogarcia) [reply packages/redshift-types/src/constants/general.ts]: sounds good now?

**Pull request merged**